config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+
```

Comment:
What's the reason for not following the [default way of creating routes in Rails](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default) using the `resources` syntax?

Reply:
Related to my comment about the models: if Posts belong to Users you may want to take a look at [nested resources](https://guides.rubyonrails.org/routing.html#nested-resources) 🙂